src/Browsershot.php

```diff
@@ -404,15 +404,10 @@ public function createBodyHtmlCommand(): array
         return $this->createCommand($url, 'content');
     }
 
-    public function createScreenshotCommand($targetPath = null): array
+    public function createScreenshotCommand($options = []): array
```

Comment:
As it stands this is a breaking change. This function should still be able to handle string input

Reply:
You mean that this needs backward compatibility?